config/dev.secret.example.exs

```diff
@@ -3,6 +3,11 @@
 # Copy the template to dev.secret.exs, then uncomment and fill in any sections you need.
 import Config
 
+# Appcues
+config :orbit,
+  appcues_enabled?: true,
+  appcues_id: "SOME_ID"
```

Comment:
This should be commented out by default so that copying the template doesn't start sending data to appcues unless you specifically turn it on. (I see that fullstory below is also uncommented but should be commented.)

An alternative would be to leave the `appcues_id` as `""`, and control enabling appcues by whether the id is non-nil and non-empty, instead of having a separate `_enabled?` and `_id` fields. Then it can be safetly left uncommented.

Reply:
06f21df